nservicebus/ravendb/unit-of-work.md

```diff
@@ -14,31 +14,55 @@ Share the session between the message handler(s) and the actual unit of work imp
 
 Instead, beginning with NServiceBus version 3, use the new [support for child containers](/nservicebus/containers/child-containers.md). This means that all dependencies configured as a single call effectively become static within the context of one transport message.
 
-To resolve a RavenDB document session from the container, add the following configuration (StructureMap is used but any of the other containers except Spring and Unity would work):
-
+To resolve a RavenDB document session from the container, add the following configuration (StructureMap is used but any of the other containers except Spring and Unity would work):  
```

Comment:
Blog is back now

I'll rework the page to not point to link to my blog since we have all
content we need on docs.particular.net

On Mon, Aug 17, 2015 at 10:32 AM, pashute <notifications@github.com> wrote:

> In nservicebus/ravendb/unit-of-work.md
> <https://github.com/Particular/docs.particular.net/pull/747#discussion_r37167215>
> :
>
> > @@ -14,31 +14,55 @@ Share the session between the message handler(s) and the actual unit of work imp
> >
> >  Instead, beginning with NServiceBus version 3, use the new [support for child containers](/nservicebus/containers/child-containers.md). This means that all dependencies configured as a single call effectively become static within the context of one transport message.
> >
> > -To resolve a RavenDB document session from the container, add the following configuration (StructureMap is used but any of the other containers except Spring and Unity would work):
> > -
> > +To resolve a RavenDB document session from the container, add the following configuration (StructureMap is used but any of the other containers except Spring and Unity would work):
>
> the links to andreas' blog don't work anymore...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/docs.particular.net/pull/747/files#r37167215>
> .
>